src/app/_components/Label.js

```diff
@@ -0,0 +1,18 @@
+import PropTypes from 'prop-types';
+import { cn } from '@/utils/cn';
+
+export function Label({ bgColor, textColor, icon, text }) {
+  return (
+    <div className={cn('inline-flex items-center justify-center gap-1 rounded-3xl px-3 py-1', bgColor)}>
+      {icon}
+      <p className={cn('text-c3', textColor)}>{text}</p>
+    </div>
+  );
+}
```

Comment:
I would recomend to use more general names for the class names props. For example: 
```suggestion
export function Label({ className, textClassName, icon, text }) {
  return (
    <div className={cn('inline-flex items-center justify-center gap-1 rounded-3xl px-3 py-1', className)}>
      {icon}
      <p className={cn('text-c3', textClassName)}>{text}</p>
    </div>
  );
}
```

Reply:
Changed it